src/components/account-header/account-header.yaml

```diff
@@ -87,6 +87,7 @@ examples:
       classes: 'the-header-classes'
   visualRegressionTests:
     backstopScenarioOptions:
+      misMatchThreshold: 0.1
```

Comment:
I'm guessing this was introduced as there is probably a pixel that is out of place when hovering occurs.